package.json

```diff
@@ -9,7 +9,9 @@
     "wpe-build": "npm run build",
     "dev": "next dev",
     "format": "yarn lint --fix",
-    "lint": "eslint",
+    "format:no-path": "yarn lint:no-path --fix",
+    "lint": "eslint .",
+    "lint:nopath": "eslint",
```

Comment:
Little typo: this should be `lint:no-path` or the _line 12_  `"format:no-path": "yarn lint:nopath --fix"`

Reply:
it took me forever to see what you were pointing out 😂  thanks, good catch

Reply:
hahah sorry for that, the explanation is actually confusing 😅